kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProduceAction.java

```diff
@@ -131,15 +127,16 @@ public void produce(
 
   private CompletableFuture<ProduceResponse> produce(
       String clusterId, String topicName, ProduceRequest request, ProduceController controller) {
+    Instant requestStart = Instant.now();
```

Comment:
`Instant.now()` is not the best way to measure elapsed time on the same machine, as it ends up using a system clock that measures wall time (which is not monotonic). Let's use `System.nanoTime()` instead.